.kokoro/tests/run_tests.sh

```diff
@@ -90,6 +90,33 @@ for file in **/requirements.txt; do
     nox -s "$RUN_TESTS_SESSION"
     EXIT=$?
 
+    # If we're running system tests, send the test log to the Build Cop Bot.
+    # See https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop.
+    if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* ]]; then
```

Comment:
Is this system-tests directory something that's on the kokoro end or is it searching for it here? 

If searching here, it won't find it - we restructured the kokoro config files to be broken down by python version and then into continuous, periodic, presubmit, and lint, unlike before when we had system tests by product

Reply:
Good catch. Switched to continuous. Let me know if you'd prefer periodic.

Reply:
Sweet. They're all going to fail right away - we're slowly but surely resolving all failures within.